scripts/metadata-export-search.md

```diff
@@ -0,0 +1,41 @@
+# Metadata-Export-Search Script
+<!-- TOC -->
+* [Metadata-Export-Search Script](#metadata-export-search-script)
+  * [What it does?](#what-it-does)
+  * [When we use it?](#when-we-use-it)
+  * [How it Works?](#how-it-works)
+  * [Parameters](#parameters)
+  * [Conclusion/QuickGuide:](#conclusionquickguide)
+<!-- TOC -->
+## What it does?
+
+The Metadata-Export-Search script exports metadata from a discovery search in DSpace in CSV format.
+
+## When we use it?
+
+The Metadata-Export-Search script is used to export metadata from a discovery search in DSpace. This script can be
+useful for creating reports or generating metadata exports based on specific search criteria.
```

Comment:
This can also be useful for Batch Metadata Editing.  So, we may want to link it in as well. https://wiki.lyrasis.org/display/DSDOC7x/Batch+Metadata+Editing

Reply:
@Sondissimo You should says something like This export can be used as starting point to perform offline bulk changes to the record via the [Bach Metadata Editing feature](https://wiki.lyrasis.org/display/DSDOC7x/Batch+Metadata+Editing)